packages/manager/apps/key-management-service/src/public/translations/key-management-service/common/Messages_fr_FR.json

```diff
@@ -0,0 +1,4 @@
+{
+  "clipboard_copy_success": "Copié !",
+  "clipboard_copy_error": "Erreur copie."
```

Comment:
Can you use the same file everywhere to avoid duplicating content  for translation